packages/web/e2e/pages/portfolio-page.ts

```diff
@@ -61,4 +64,10 @@ export class PortfolioPage extends BasePage {
       await this.page.waitForTimeout(1000);
     }
   }
+
+  async searchForToken(tokenName: string) {
+    await this.searchInput.fill(tokenName);
+    // we expect that after 2 seconds tokens are loaded and any failure after this point should be considered a bug.
+    await this.page.waitForTimeout(2000);
```

Comment:
Would there be a clear feedback about why the test failed? If not, is there anywhere we could add a comment or a log?